webhooks-extension/endpoints/sink.go

```diff
@@ -0,0 +1,329 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+		http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package endpoints
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	restful "github.com/emicklei/go-restful"
+	v1alpha1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	gh "gopkg.in/go-playground/webhooks.v3/github"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const gitServerLabel = "gitServer"
+const gitOrgLabel = "gitOrg"
+const gitRepoLabel = "gitRepo"
+const githubEventParameter = "Ce-Github-Event"
```

Comment:
Sceptical about this, we've had this change on us a lot (sometimes being Ce-X) and had it as a variable we pass in (through the yaml)..